projects/angular/src/emphasis/_labels.clarity.scss

```diff
@@ -196,76 +94,18 @@
       text-decoration: none;
     }
 
-    &.clickable {
-      @include mixins.css-var(
-        font-size,
-        clr-label-clickable-font-size,
-        label-variables.$clr-label-font-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-weight,
-        clr-label-clickable-font-weight,
-        label-variables.$clr-label-font-weight,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        letter-spacing,
-        clr-label-clickable-letter-spacing,
-        label-variables.$clr-label-letter-spacing,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-label-clickable-line-height,
-        // line-height needed to be adjusted to force center alignment
-        label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px,
-        variables.$clr-use-custom-properties
-      );
-    }
-
     &.clickable:not(.disabled) {
       &:hover {
         text-decoration: none;
-        @include mixins.css-var(
-          background,
-          clr-label-bg-hover-color,
-          label-variables.$clr-label-bg-hover-color,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          color,
-          clr-label-hover-color,
-          label-variables.$clr-label-font-color-light,
-          variables.$clr-use-custom-properties
-        );
+        background: label-variables.$clr-label-bg-hover-color;
+        color: label-variables.$clr-label-hover-color;
       }
 
       &:active {
-        @include mixins.css-var(
-          color,
-          clr-label-active-color,
-          label-variables.$clr-label-font-color-light,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          background,
-          clr-label-bg-active-color,
-          label-variables.$clr-label-bg-hover-color,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          box-shadow,
-          clr-label-box-shadow,
-          0 variables.$clr_baselineRem_1px 0 0 label-variables.getLabelColor(gray, bg-color) inset,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          transform,
-          clr-label-pressed-transform,
-          translateY(0.5px),
-          variables.$clr-use-custom-properties
-        );
+        color: label-variables.$clr-label-active-color;
+        background: label-variables.$clr-label-bg-active-color;
+        box-shadow: label-variables.$clr-label-box-shadow;
```

Comment:
`--clr-label-box-shadow` property is removed. Probably we need to delete this `box-shadow` rule?
```suggestion
```